test/metabase/query_processor/streaming_test.clj

```diff
@@ -36,86 +36,96 @@
 (defn- basic-actual-results* [export-format query]
   (maybe-remove-checksum (streaming.test-util/process-query-basic-streaming export-format query)))
 
-(deftest basic-streaming-test
+(defn- basic-streaming-test-query []
```

Comment:
I split the test below into four separate tests because it was really annoying to debug and they were logically separate test cases anyway